### ISSUE TITLE
Route color

### ISSUE THREAD

**Issue by ivanjolic95** [asennikov/ember-g-map#36]
While working on #35 I tried to add color option to route component but I couldn't make it work.

I suppose this should be added to rendererOptions for direction renderer in g-map-route component:

``` 
let color = this.get('color');
...
polylineOptions: {
  strokeColor: color
}
```

For some reason it didn't work. Can you please look into it? Thanks!

**Comment by asennikov** [comment]
Yeah, will look into this, would be definitely good to be able to change color of those lines.

**Comment by asennikov** [comment]
Looks like I was able to do that. Check it out at https://github.com/asennikov/ember-g-map/pull/38, I'm going to add some tests and release it soon.

**Comment by ivanjolic95** [comment]
Thanks! I've added a basic "waypoints" option to route (passing an array of waypoints as it's specified in maps api) .

I'd like to extract them into g-map-waypoint component though, so they can be added within a g-map-route block. What do you think of that approach?

If it's okay I'll make a pull request next week.

**Comment by asennikov** [comment]
Sounds perfect!

**Comment by asennikov** [comment]
Route colors (and some other polyline options are merged) are available in `0.0.16`, so I'm closing this issue.

Waiting for that waypoints PR though :wink:
